packages/validator/test/cli-validator/mock-files/just-warn-config-override.json

```diff
@@ -3,8 +3,8 @@
     "operations": {
       "no_operation_id": "off"
     },
-    "schemas": {
-      "no_schema_description": "error"
+    "responses": {
+      "inline_response_schema": "error"
```

Comment:
had to pick a different rule for testing since no_schema_description no longer "works" :)